src/realm/array_packed.hpp

```diff
@@ -0,0 +1,309 @@
+/*************************************************************************
```

Comment:
This files contains functions that are needed by the bptree implementation. It might be that they are not all needed, but it seemed logical just to move all "direct access functions".